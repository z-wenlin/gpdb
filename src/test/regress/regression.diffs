diff -I HINT: -I CONTEXT: -I GP_IGNORE: -U3 /home/gpadmin/workspace/gpdb6/src/test/regress/expected/partition_prune.out /home/gpadmin/workspace/gpdb6/src/test/regress/results/partition_prune.out
--- /home/gpadmin/workspace/gpdb6/src/test/regress/expected/partition_prune.out	2023-07-07 11:35:09.025409269 +0800
+++ /home/gpadmin/workspace/gpdb6/src/test/regress/results/partition_prune.out	2023-07-07 11:35:09.259411165 +0800
@@ -3254,7 +3254,7 @@
                            Filter: (b = $0)
                      ->  Seq Scan on ab_a2_b3 (never executed)
                            Filter: (b = $0)
-         ->  Materialize (actual rows=1 loops=1)
+         ->  Materialize (actual rows=1 loops=2)
                ->  Seq Scan on ab_a1_b1 (actual rows=1 loops=1)
    ->  Nested Loop (actual rows=1 loops=1)
          ->  Broadcast Motion 3:3  (slice3; segments: 3) (actual rows=1 loops=1)
@@ -3265,7 +3265,7 @@
                            Filter: (b = $0)
                      ->  Seq Scan on ab_a2_b3 (never executed)
                            Filter: (b = $0)
-         ->  Materialize (actual rows=1 loops=1)
+         ->  Materialize (actual rows=1 loops=2)
                ->  Seq Scan on ab_a1_b2 (actual rows=1 loops=1)
    ->  Nested Loop (actual rows=1 loops=1)
          ->  Broadcast Motion 3:3  (slice4; segments: 3) (actual rows=1 loops=1)
@@ -3276,7 +3276,7 @@
                            Filter: (b = $0)
                      ->  Seq Scan on ab_a2_b3 (never executed)
                            Filter: (b = $0)
-         ->  Materialize (actual rows=1 loops=1)
+         ->  Materialize (actual rows=1 loops=2)
                ->  Seq Scan on ab_a1_b3 (actual rows=1 loops=1)
 GP_IGNORE:(40 rows)
 
