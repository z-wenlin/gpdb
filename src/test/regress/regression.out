Optimizer disabled. Using planner answer files
Resource group disabled. Using default answer files
test test_setup                   ... ok           35 ms (diff   48 ms)
test tablespace                   ... ok         1717 ms (diff  103 ms)
parallel group (20 tests):  regproc pg_lsn text txid varchar oid name char int2 float4 boolean uuid money int4 float8 bit int8 enum numeric rangetypes
     boolean                      ... ok          751 ms (diff   80 ms)
     char                         ... ok          469 ms (diff  103 ms)
     name                         ... ok          410 ms (diff   58 ms)
     varchar                      ... ok          367 ms (diff  102 ms)
     text                         ... ok          291 ms (diff   76 ms)
     int2                         ... ok          552 ms (diff   65 ms)
     int4                         ... ok          899 ms (diff   71 ms)
     int8                         ... ok         1094 ms (diff  103 ms)
     oid                          ... ok          402 ms (diff   51 ms)
     float4                       ... ok          655 ms (diff  107 ms)
     float8                       ... ok         1080 ms (diff  134 ms)
     bit                          ... ok         1083 ms (diff   92 ms)
     numeric                      ... ok         1978 ms (diff  191 ms)
     txid                         ... ok          346 ms (diff   66 ms)
     uuid                         ... ok          758 ms (diff   54 ms)
     enum                         ... ok         1299 ms (diff   89 ms)
     money                        ... ok          808 ms (diff   76 ms)
     rangetypes                   ... ok         2038 ms (diff  152 ms)
     pg_lsn                       ... ok          223 ms (diff   57 ms)
     regproc                      ... ok          179 ms (diff   69 ms)
parallel group (19 tests):  tstypes time lseg timetz line date path circle macaddr macaddr8 point interval inet numerology timestamp box polygon timestamptz strings
     strings                      ... ok         1036 ms (diff  181 ms)
     numerology                   ... ok          773 ms (diff   55 ms)
     point                        ... ok          579 ms (diff   71 ms)
     lseg                         ... ok          296 ms (diff   49 ms)
     line                         ... ok          321 ms (diff   50 ms)
     box                          ... ok          848 ms (diff   82 ms)
     path                         ... ok          383 ms (diff   49 ms)
     polygon                      ... ok          939 ms (diff   66 ms)
     circle                       ... ok          388 ms (diff   52 ms)
     date                         ... ok          336 ms (diff  140 ms)
     time                         ... ok          264 ms (diff   52 ms)
     timetz                       ... ok          310 ms (diff   57 ms)
     timestamp                    ... ok          832 ms (diff  156 ms)
     timestamptz                  ... ok         1002 ms (diff  225 ms)
     interval                     ... ok          590 ms (diff  107 ms)
     inet                         ... ok          701 ms (diff   99 ms)
     macaddr                      ... ok          454 ms (diff   55 ms)
     macaddr8                     ... ok          566 ms (diff   69 ms)
     tstypes                      ... ok          153 ms (diff  128 ms)
parallel group (9 tests):  comments misc_sanity expressions type_sanity oidjoins regex horology opr_sanity geometry
     geometry                     ... ok         1377 ms (diff  352 ms)
     horology                     ... ok          577 ms (diff  261 ms)
     regex                        ... ok          539 ms (diff   83 ms)
     oidjoins                     ... ok          478 ms (diff  134 ms)
     type_sanity                  ... ok          317 ms (diff   99 ms)
     opr_sanity                   ... ok          851 ms (diff  204 ms)
     misc_sanity                  ... ok          108 ms (diff   55 ms)
     comments                     ... ok           19 ms (diff   48 ms)
     expressions                  ... ok          222 ms (diff   54 ms)
test create_function_1            ... ok          121 ms (diff   53 ms)
test create_type                  ... ok          173 ms (diff   63 ms)
test create_table                 ... ok         3521 ms (diff  150 ms)
test create_function_2            ... ok           71 ms (diff   51 ms)
parallel group (5 tests):  copyselect insert_conflict copydml copy insert
     copy                         ... ok         2849 ms (diff   67 ms)
     copyselect                   ... ok          380 ms (diff   55 ms)
     copydml                      ... ok         1750 ms (diff   53 ms)
     insert                       ... ok         5692 ms (diff  124 ms)
     insert_conflict              ... ok         1694 ms (diff  107 ms)
parallel group (3 tests):  create_operator create_procedure create_misc
     create_misc                  ... ok          462 ms (diff   57 ms)
     create_operator              ... ok          159 ms (diff   64 ms)
     create_procedure             ... ok          217 ms (diff   59 ms)
parallel group (5 tests):  index_including_gist create_index_spgist index_including create_view create_index
     create_index                 ... ok         4222 ms (diff  242 ms)
     create_index_spgist          ... ok         1653 ms (diff  138 ms)
     create_view                  ... ok         3416 ms (diff  175 ms)
     index_including              ... ok         2124 ms (diff   76 ms)
     index_including_gist         ... ok         1017 ms (diff   58 ms)
test gp_gin_index                 ... ok          269 ms (diff  123 ms)
parallel group (16 tests):  errors hash_func infinite_recurse create_cast roleattributes typed_table create_aggregate create_function_3 create_am select drop_if_exists vacuum constraints inherit updatable_views triggers
     create_aggregate             ... ok         1214 ms (diff   71 ms)
     create_function_3            ... ok         1240 ms (diff   70 ms)
     create_cast                  ... ok          308 ms (diff   49 ms)
     constraints                  ... ok         4862 ms (diff  101 ms)
     triggers                     ... ok         9485 ms (diff  275 ms)
     select                       ... ok         1602 ms (diff  108 ms)
     inherit                      ... ok         9043 ms (diff  238 ms)
     typed_table                  ... ok         1087 ms (diff   56 ms)
     vacuum                       ... ok         2993 ms (diff   71 ms)
     drop_if_exists               ... ok         1763 ms (diff   70 ms)
     updatable_views              ... ok         9314 ms (diff  277 ms)
     roleattributes               ... ok          908 ms (diff   68 ms)
     create_am                    ... ok         1562 ms (diff   71 ms)
     hash_func                    ... ok          161 ms (diff   70 ms)
     errors                       ... ok           47 ms (diff   76 ms)
     infinite_recurse             ... ok          223 ms (diff   46 ms)
test disable_autovacuum           ... ok           27 ms (diff   46 ms)
test vacuum_stats                 ... ok           83 ms (diff   46 ms)
test enable_autovacuum            ... ok           25 ms (diff   46 ms)
parallel group (17 tests):  select_distinct_on select_having random delete case select_implicit hash_index select_into btree_index union select_distinct portals subselect arrays join update aggregates
     select_into                  ... ok         2327 ms (diff   57 ms)
     select_distinct              ... ok         2794 ms (diff   87 ms)
     select_distinct_on           ... ok          182 ms (diff   49 ms)
     select_implicit              ... ok         1062 ms (diff  198 ms)
     select_having                ... ok          367 ms (diff   50 ms)
     subselect                    ... ok         3948 ms (diff  165 ms)
     union                        ... ok         2763 ms (diff  115 ms)
     case                         ... ok         1010 ms (diff   70 ms)
     join                         ... ok         7911 ms (diff  451 ms)
     aggregates                   ... ok        12596 ms (diff  261 ms)
     random                       ... ok          420 ms (diff   48 ms)
     portals                      ... ok         3181 ms (diff  129 ms)
     arrays                       ... ok         5077 ms (diff  208 ms)
     btree_index                  ... ok         2332 ms (diff   77 ms)
     hash_index                   ... ok         1835 ms (diff   61 ms)
     update                       ... ok         9599 ms (diff  124 ms)
     delete                       ... ok          810 ms (diff   47 ms)
test namespace                    ... ok          133 ms (diff   52 ms)
parallel group (20 tests):  init_privs drop_operator security_label tablesample password lock gin replica_identity object_address spgist gist collate identity matview generated join_hash brin privileges rowsecurity groupingsets
     brin                         ... ok         9345 ms (diff   89 ms)
     gin                          ... ok         2797 ms (diff   49 ms)
     gist                         ... ok         3178 ms (diff   68 ms)
     spgist                       ... ok         3005 ms (diff   50 ms)
     privileges                   ... ok        10514 ms (diff  218 ms)
     init_privs                   ... ok          201 ms (diff   45 ms)
     security_label               ... ok         1165 ms (diff   48 ms)
     collate                      ... ok         3366 ms (diff   93 ms)
     matview                      ... ok         6730 ms (diff   93 ms)
     lock                         ... ok         2554 ms (diff   59 ms)
     replica_identity             ... ok         2798 ms (diff   63 ms)
     rowsecurity                  ... ok        11555 ms (diff  348 ms)
     object_address               ... ok         2811 ms (diff  200 ms)
     tablesample                  ... ok         2266 ms (diff   74 ms)
     groupingsets                 ... ok        12274 ms (diff  187 ms)
     drop_operator                ... ok          428 ms (diff   48 ms)
     password                     ... ok         2332 ms (diff   55 ms)
     identity                     ... ok         4338 ms (diff   74 ms)
     generated                    ... ok         7649 ms (diff  117 ms)
     join_hash                    ... ok         8335 ms (diff  122 ms)
parallel group (11 tests):  dbsize async sysviews alter_operator misc_functions tsrf tidscan tid alter_generic misc create_table_like
     create_table_like            ... ok         2263 ms (diff   86 ms)
     alter_generic                ... ok         1136 ms (diff  104 ms)
     alter_operator               ... ok          211 ms (diff   55 ms)
     misc                         ... ok         1251 ms (diff   81 ms)
     async                        ... ok           56 ms (diff   48 ms)
     dbsize                       ... ok           23 ms (diff   57 ms)
     misc_functions               ... ok          254 ms (diff   64 ms)
     sysviews                     ... ok          148 ms (diff   53 ms)
     tsrf                         ... ok          430 ms (diff   91 ms)
     tid                          ... ok          507 ms (diff   50 ms)
     tidscan                      ... ok          444 ms (diff   66 ms)
parallel group (4 tests):  amutils psql_crosstab psql stats_ext
     psql                         ... ok         1326 ms (diff  366 ms)
     psql_crosstab                ... ok          219 ms (diff   60 ms)
     amutils                      ... ok           95 ms (diff   62 ms)
     stats_ext                    ... ok         2911 ms (diff  129 ms)
test select_parallel              ... ok         6168 ms (diff  134 ms)
test write_parallel               ... ok          114 ms (diff   50 ms)
parallel group (2 tests):  subscription publication
     publication                  ... ok          550 ms (diff   64 ms)
     subscription                 ... ok          118 ms (diff   59 ms)
parallel group (16 tests):  advisory_lock portals_p2 tsdicts combocid xmlmap functional_deps indirect_toast select_views guc equivclass dependency tsearch window cluster bitmapops foreign_data
     select_views                 ... ok         1358 ms (diff  161 ms)
     portals_p2                   ... ok          269 ms (diff   54 ms)
     cluster                      ... ok         3006 ms (diff   87 ms)
     dependency                   ... ok         1678 ms (diff   56 ms)
     guc                          ... ok         1439 ms (diff   99 ms)
     bitmapops                    ... ok         4254 ms (diff   83 ms)
     combocid                     ... ok          794 ms (diff   56 ms)
     tsearch                      ... ok         2276 ms (diff  196 ms)
     tsdicts                      ... ok          708 ms (diff   87 ms)
     foreign_data                 ... ok         4651 ms (diff  210 ms)
     window                       ... ok         2418 ms (diff  304 ms)
     xmlmap                       ... ok          829 ms (diff  133 ms)
     functional_deps              ... ok         1193 ms (diff   68 ms)
     advisory_lock                ... ok          247 ms (diff   65 ms)
     indirect_toast               ... ok         1274 ms (diff   56 ms)
     equivclass                   ... ok         1619 ms (diff   79 ms)
parallel group (6 tests):  jsonpath_encoding json_encoding jsonpath jsonb_jsonpath json jsonb
     json                         ... ok          498 ms (diff  216 ms)
     jsonb                        ... ok          885 ms (diff  367 ms)
     json_encoding                ... ok           18 ms (diff   62 ms)
     jsonpath                     ... ok           35 ms (diff  104 ms)
     jsonpath_encoding            ... ok           14 ms (diff   56 ms)
     jsonb_jsonpath               ... ok           92 ms (diff  171 ms)
parallel group (17 tests):  prepare limit conversion returning xml plancache rowtypes sequence polymorphism copy2 temp with truncate rangefuncs domain plpgsql alter_table
     plancache                    ... ok         2687 ms (diff   75 ms)
     limit                        ... ok          357 ms (diff   68 ms)
     plpgsql                      ... ok        12200 ms (diff  416 ms)
     copy2                        ... ok         2999 ms (diff   83 ms)
     temp                         ... ok         3111 ms (diff   74 ms)
     domain                       ... ok         7991 ms (diff  133 ms)
     rangefuncs                   ... ok         4358 ms (diff  191 ms)
     prepare                      ... ok          275 ms (diff   58 ms)
     conversion                   ... ok          397 ms (diff   48 ms)
     truncate                     ... ok         4259 ms (diff   85 ms)
     alter_table                  ... ok        18934 ms (diff  412 ms)
     sequence                     ... ok         2900 ms (diff   98 ms)
     polymorphism                 ... ok         2962 ms (diff  154 ms)
     rowtypes                     ... ok         2715 ms (diff  125 ms)
     returning                    ... ok         1602 ms (diff   68 ms)
     with                         ... ok         3385 ms (diff  198 ms)
     xml                          ... ok         1919 ms (diff  469 ms)
test gp_foreign_data              ... ok          186 ms (diff   47 ms)
parallel group (7 tests):  hash_part reloptions partition_info partition_aggregate partition_join partition_prune indexing
     partition_join               ... ok         5125 ms (diff  211 ms)
     partition_prune              ... FAILED     5983 ms (diff  939 ms)
     reloptions                   ... ok          643 ms (diff   63 ms)
     hash_part                    ... ok          314 ms (diff   53 ms)
     indexing                     ... ok         7468 ms (diff  162 ms)
     partition_aggregate          ... ok         2591 ms (diff  164 ms)
     partition_info               ... ok          839 ms (diff   69 ms)
test event_trigger                ... ok         1380 ms (diff   95 ms)
test fast_default                 ... ok         1767 ms (diff  106 ms)
test constraints_check            ... ok           81 ms (diff   46 ms)
test test_setup                   ... ok           38 ms (diff   47 ms)
test join_gp                      ... ok         3769 ms (diff  291 ms)
test disable_autovacuum           ... ok           26 ms (diff   48 ms)
test motion_deadlock              ... ok        18261 ms (diff   96 ms)
